### PR TITLE
Add en-US spell checker

### PR DESCRIPTION
**Pull request by mheap**
## Description

We write the docs in US English and enforce this at build time with a CI check. You can run the check locally using the following command:

```
./node_modules/.bin/mdspell -r -n -a --en-us '_documentation/**/*.md' '_partials/*.md' '_partials/**/*.md' '_modals/**/*.md' '_tutorials/**.md'
```

If there is a word that isn't in the dictionary but is correct to use, add it to the `.spelling` file (there's a lot of exceptions in there, including `Nexmo`!)

## Deploy Notes

N/A

### CODE REVIEW

File: _documentation/conversation/concepts/conversation.md [hunk 1]
```diff
@@ -421,10 +421,10 @@ You can see the Events that occurred during this transient Conversation are as f
 Event ID | Member ID | Event type | Description
 ----|----|----|----
 1 | MEM-f44c872e-cba9-444f-88ae-0bfa630865a6 | `member:joined` | Alice to Nexmo Number
-2 | MEM-25ccda92-839d-4ac6-a7b2-de310224878b | `sip:ringing` | Nexmo to Bob
```
Comment (mheap) [inline _documentation/conversation/concepts/conversation.md]: This is an error that I need to fix. We should have two different member IDs here

File: _documentation/conversation/concepts/conversation.md [hunk 2]
```diff
@@ -421,10 +421,10 @@ You can see the Events that occurred during this transient Conversation are as f
...
-3 | MEM-25ccda92-839d-4ac6-a7b2-de310224878b | `sip:answered` | Bob answered
-4 | MEM-25ccda92-839d-4ac6-a7b2-de310224878b | `member:joined` | Bob joined the Conversation
-5 | MEM-25ccda92-839d-4ac6-a7b2-de310224878b | `member:media` | Bob audio setup
+2 | MEM-f44c872e-cba9-444f-88ae-0bfa630865a6 | `sip:ringing` | Nexmo to Bob
+3 | MEM-f44c872e-cba9-444f-88ae-0bfa630865a6 | `sip:answered` | Bob answered
+4 | MEM-f44c872e-cba9-444f-88ae-0bfa630865a6 | `member:joined` | Bob joined the Conversation
+5 | MEM-f44c872e-cba9-444f-88ae-0bfa630865a6 | `member:media` | Bob audio setup
```
Comment (tbedford) [inline _documentation/conversation/concepts/conversation.md]: ??

File: _documentation/voice/voice-api/ncco-reference.md [hunk 3]
```diff
@@ -112,7 +112,7 @@ You can use the following options to control a *conversation* action:
 Option | Description | Required
 -- | -- | --
 `name` | The name of the Conversation room. Names are namespaced to the application level. | Yes
-`musicOnHoldUrl` | A URL to the *mp3* file to stream to participants until the conversation starts. By default the conversation starts when the first person calls the virtual number associated with your Voice app. To stream this mp3 before the moderator joins the conversation, set *startOnEnter* to *false* for all users other than the moderator. | No
+`musicOnHoldURL` | A URL to the *mp3* file to stream to participants until the conversation starts. By default the conversation starts when the first person calls the virtual number associated with your Voice app. To stream this mp3 before the moderator joins the conversation, set *startOnEnter* to *false* for all users other than the moderator. | No
```
Comment (tbedford) [inline _documentation/voice/voice-api/ncco-reference.md]: Possibly shouldn't change depending on what the property is actually called.

**Comment by tbedford** [comment]
@mheap: Some problem, but I'm not sure what. I received an email about Heroku API limits earlier?

File: _documentation/voice/voice-api/webhook-reference.md [hunk 4]
```diff
@@ -268,7 +268,7 @@ Field | Example | Description
 
 ### Record
 
-This webhook arrives when an NCCO with a "record" action has finished. When creating a record action, you can set a different `eventURL` for this event to be sent to. This can be useful if you want to use separate code to handle this event type.
+This webhook arrives when an NCCO with a "record" action has finished. When creating a record action, you can set a different `eventUrl` for this event to be sent to. This can be useful if you want to use separate code to handle this event type.
```
Comment (tbedford) [inline _documentation/voice/voice-api/webhook-reference.md]: Here we change `eventURL` to `EventUrl` but earlier in the section `streamUrl` was changed to `streamURL`.

**Comment by mheap** [comment]
The failing build is that Travis uses a different dictionary to my local machine 🤦‍♂ 

I'm looking in to it

**Pull request merged**